bin/pic-create

```diff
@@ -23,12 +23,23 @@ this_dir=$(cd $(dirname $0) && pwd)
 
 # PIConGPU prefix path
 picongpu_prefix=$(cd $this_dir/.. && pwd)
-# files always cloned from defaults, can be overwritten by input set
-default_folder_to_copy="etc/picongpu lib"
-# if we clone an input set we copy from it
+# All files in the folder (no recusion or subfolders) will be copied from PIConGPU source to the input set.
+default_folder_to_copy="etc/picongpu"
+
+# Folder which get recusivly copied from the source input set into the destination input set
 folder_to_clone="etc/picongpu bin include/picongpu lib"
 files_to_copy="cmakeFlags executeOnClone"
 
+if [ -n "$PIC_SYSTEM_TEMPLATE_PATH" ] ; then
+    default_folder_to_copy="$default_folder_to_copy $PIC_SYSTEM_TEMPLATE_PATH"
+else
+    # Fallback to be backward compatible to the old behaviour that all content of picongpu/etc is copied into the
+    # destination input set.
+    # Fallback is only aktive if the environment variable 'PIC_SYSTEM_TEMPLATE_PATH' not exists.
+    # All subfolder will be copied.
+    full_folder_to_copy="etc/picongpu"
```

Comment:
This is already set in line 27 - why do you repeat this here?

Reply:
We handle `full_folder_to_copy` later in a separate loop to copy all subfolders.

Reply:
You are right - misread the two variable names